features/providers/step_definitions/civil_journey_steps.rb

```diff
@@ -108,6 +108,10 @@
   expect(page).to have_content('Check your answers')
 end
 
+Then('I should be on the {string} page') do |title|
```

Comment:
There are a number of steps defined with a very similar structure to this. In fact you use one on line 241 of `civil_application_journey.feature`: "Then I should be on the Applicant page".

I'd suggest a slight rewording that I think will make it a little clearer what is being match here:
```
Then('I should be on a page showing {string}') do |title|
```
The line in the feature then becomes:
```
Then I should be on a page showing "Contact us"
```
We could probably also do with a Tech Debt ticket to remove all steps using the page specific form, and replace them with your more generic version.